Create ui-c8y-1020-40-0-avoid-unnecessary-unavailability-alarms-for-lw-m-2-m-device-graft.md (#2550)
SoftwareAG/c8y-docs: content/protocol-integration/lwm2m-bundle/device-details.md

  * Timestamp resources
  * Objects tab behavior (keep old values)
  * Disable the default behavior for object instances of objects 3 (device), 4 (connectivity monitoring) and 6 (location)

>   * Automatic setting of required interval

Reviewer: The explanation should be added here 
When enabled, the LWM2M agent automatically sets the interval to registration lifetime plus 2 minutes. If disabled, the user can define a required interval for the device on the **Info** tab in the device details. If set to default, the default behavior of the LWM2M service will be used.

Reviewer 2: Added.

>   * Automatic setting of required interval (When enabled, the LWM2M agent automatically sets the interval to registration lifetime plus 2 minutes. When false, the user can define a required interval for the device on the Info tab in the device details. If value is not provided ("null"), the default behavior of the LWM2M service will be used.)

Reviewer: ```suggestion
  * Automatic setting of required interval 
    * When enabled, the LWM2M service automatically sets the interval to registration lifetime plus 2 minutes. If disabled, the user can define a required interval for the device on the **Info** tab in the device details. If set to default, the default behavior of the LWM2M service will be used.</p>
```
